crates/wash-lib/src/component.rs

```diff
@@ -62,12 +62,12 @@ pub async fn scale_component(
     }: ScaleComponentArgs<'_>,
 ) -> Result<ComponentScaledInfo> {
     // If timeout isn't supplied, override with a longer timeout for starting component
-    let timeout_ms = timeout_ms.unwrap_or(DEFAULT_START_ACTOR_TIMEOUT_MS);
+    let timeout_ms = timeout_ms.unwrap_or(DEFAULT_START_COMPONENT_TIMEOUT_MS);
 
     // Create a receiver to use with the client
     let mut receiver = client
         .events_receiver(vec![
-            "actor_scaled".to_string(),
+            "actor_scaled".to_string(), // NOTE(nitame): Is these events_types still relevant?
```

Comment:
Just calling this out here if @brooksmtownsend confirms that we can remove these now

Reply:
Yep confirmed, we can remove the `actor_scaled` event since it's not emitted by post-1.0 hosts. All `component_scaled` now 🚀